warehouse/manage/views.py

```diff
@@ -551,6 +552,31 @@ def manage_project_roles(project, request, _form_class=CreateRoleForm):
                     submitted_from=request.remote_addr,
                 ),
             )
+
+            owners = (
+                request.db.query(Role)
+                .join(User)
+                .filter(Role.role_name == 'Owner', Role.project == project)
+            )
+            owner_emails = [owner.email for owner in owners]
+            owner_emails.remove(request.user.email)
+
+            send_collaborator_added_email(
+                request,
+                user,
+                request.user,
+                project.name,
+                form.role_name.data,
+                owner_emails
+            )
+
+            send_added_as_collaborator_email(request,
+                                             request.user,
+                                             project.name,
+                                             form.role_name.data,
+                                             user.email
+                                             )
```

Comment:
This is a nitpick, but could you make the indentation here consistent? (e.g. like the call to `send_collaborator_added_email` above)